azext_edge/tests/edge/support/create_bundle_int/test_auto_int.py

```diff
@@ -113,8 +113,11 @@ def _get_expected_services(
         expected_services = OpsServiceType.list()
         expected_services.remove(OpsServiceType.auto.value)
         expected_services.append("otel")
-        # device registry folder will not be created if there are no device registry resources
```

Comment:
will the device registry folder be created even if there are no assets/asset endpoints now?

Reply:
the folder should only create when there is crd instance, let me double check

Reply:
checked it is created only when there is crd instance